packages/scss/src/components/_chips.scss

```diff
@@ -1,19 +1,36 @@
 .chip {
-	background: _color("primary", "lightest");
-	border-radius: 1rem;
+	background: _color("primary", "700");
+	border-radius: _theme("commons.border.radius");
+	color:  _color("primary", "text");
 	display: inline-block;
 	margin: 0 .2rem .3rem 0;
 	padding: .15rem 1.8rem .15rem .75rem;
 	position: relative;
 	vertical-align: middle;
+	&:hover, &:focus {
+		background: _color("primary", "600");
+	}
+	&:focus {
+		box-shadow: 0 0 1px 2px _color("primary", "400");
+		outline: none;
+	}
+	&.is-disabled {
+		background: _color("primary", "500");
+		color: _color("primary", "300");
+		pointer-events: none;
```

Comment:
Avec ça tu empêches la fermeture via le bouton non ?

Reply:
english

Reply:
Isn't it weird to have a disable state on a chip ? It feels more like a readOnly and it shouldn't blocks pointer events, but just remove the chip kill. Keep the pointer events to allow text selection here